src/qthotkey/gui.cc

```diff
@@ -0,0 +1,334 @@
+/*
+ *  This file is part of audacious-hotkey plugin for audacious
+ *
+ *  Copyright (C) 2020 i.Dark_Templar <darktemplar@dark-templar-archives.net>
+ *  Copyright (c) 2007 - 2008  Sascha Hlusiak <contact@saschahlusiak.de>
+ *  Name: gui.c
+ *  Description: gui.c
+ *
```

Comment:
nit: this is copy paste error (file is .cc), I'd delete these useless Name: and Description: lines.